SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -399,6 +399,8 @@ private string getGluwacoinTransactionSignature(ECurrency currency, string amoun
 
             ABIEncode abiEncode = new ABIEncode();
             byte[] messageHash = abiEncode.GetSha3ABIEncodedPacked(
+                new ABIValue("unit8", 3),// Domain 3 for transfer
```

Comment:
1. move 3 and 4 into constants so they don't look like some magic numbers.
2. This signature format is only for USDC-G, right @kevinnguyen17 ? For other currencies it's different?

Reply:
Yes, it is for USDC-G for now. the newly developed KRWC-G & sKRWC-G are the same as USDC-G. The rest (NGN-G, sUSDC-G, sNGN-G) will stay as-is until we have a plan to upgrade their contracts.